semcore/feedback-form/CHANGELOG.md

```diff
@@ -2,7 +2,11 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
-## [6.37.1] - 2024-12-05
```

Comment:
I think you should stay it as is and just add 6.38.0 for you a11y changes.
Also maybe let's write some points like `added aria-haspopup for controls with errors`, `added dynamic role slider or img depends on readonly property for Slider` and so on

Reply:
Added